package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "booking-plugin",
+  "version": "0.1.0",
+  "description": "Booking plugin application",
+  "scripts": {
+    "watch:css": "./node_modules/.bin/webpack --watch",
+    "build:css": "./node_modules/.bin/webpack -p",
+    "build": "...",
+    "postinstall": "cp ./node_modules/booking-js/dist/js/app.min.js ./dist/app.min.js"
+  },
+  "devDependencies": {
+    "booking-js": "git@github.com:RebelCode/bookings-js.git#11bc8c09ea0d6f52ed5e587297c8ddb3bf456fcc",
```

Comment:
If possible, declare dependency on a tag instead of commit. Even better - use a constraint according to SemVer.